sample/src/commonMain/kotlin/com/swmansion/kmpmaps/sample/MapsScreen.kt

```diff
@@ -176,6 +208,43 @@ internal fun MapsScreen() {
                         )
                     },
                 )
+                Row(
+                    modifier = Modifier.fillMaxWidth().padding(horizontal = 16.dp),
+                    horizontalArrangement = Arrangement.spacedBy(8.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                ) {
+                    Text("GeoJSON")
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier.clickable { showPointGeoJson = !showPointGeoJson },
+                    ) {
+                        Checkbox(
+                            checked = showPointGeoJson,
+                            onCheckedChange = { showPointGeoJson = it },
+                        )
+                        Text("Point")
+                    }
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier.clickable { showPolygonGeoJson = !showPolygonGeoJson },
+                    ) {
+                        Checkbox(
+                            checked = showPolygonGeoJson,
+                            onCheckedChange = { showPolygonGeoJson = it },
+                        )
+                        Text("Area")
+                    }
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier.clickable { showLineGeoJson = !showLineGeoJson },
+                    ) {
+                        Checkbox(
+                            checked = showLineGeoJson,
+                            onCheckedChange = { showLineGeoJson = it },
+                        )
+                        Text("Line")
+                    }
```

Comment:
I’d suggest using [filter chips](https://m3.material.io/components/chips/specs), which are more common for such use cases. Checkboxes are typically seen in vertical lists. As an additional benefit, the chips should be more compact.